Documentation/DE/Referenz/Hooks/Index.rst

```diff
@@ -385,83 +385,134 @@ Implementieren Sie die benötigten Methoden gemäß dem Interface:
 
 .. _notificationemail_de:
 
-Hooks to post process notification emails
-""""""""""""""""""""""""""""""""""""""""""
+Hooks zu den Emails der Registrierungsbenachrichtigungen
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-To use this hook, please create a class that implements the interface
-\\OliverKlee\\Seminars\\Hooks\\RegistrationEmailHookInterface. Then you need to add the following methods:
+Es gibt 4 Hooks in die Emails der Registrierungsbenachrichtigungen:
```

Comment:
I'd say "für die" instead of "in die" here. WDYT?

Reply:
It's like that in the other hook docs, too. It's an english word in a german sentence, and in English I would say "hook-in to" -> "einhaken in". But I don't really care. Less work is keep it.